### DISCUSSION TITLE
Support Needed for GitHub Campus Program Enrollment for BCS Admission

### DISCUSSION THREAD

**Discussion by pranto2024** [community/community#141296] [GitHub Education]
### Select Topic Area

General

### Body

Hello, GitHub Support Team,

I am inquiring about enrolling our institution, BCS Admission, in the GitHub Campus Program.  We are eager to take advantage of the Campus Program's resources to enhance our students' learning experiences.

BCS Admission is an individual-owned, government-approved institution with over 2,000 enrolled students. Joining the GitHub Campus Program will significantly benefit our students and help them develop critical technical skills through GitHub's educational tools.

Please guide me through the application process and inform me of any specific requirements or best practices for a successful application. Any assistance you could provide in helping us secure our place in the program would be greatly appreciated.

Thank you very much for your time and support. I look forward to hearing back from you.

Warm regards,
Md. Pranto Islam
GitHub Username: pranto2024
Email: pranto@bcsadmission.com
Institution: BCS Admission
Location: Bangladesh

**Comment by ebndev** [comment]
Hi @pranto2024 thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `GitHub Education` category, as this topic may be more relevant there and will ensure your request can be reviewed by the proper GitHub team / directed to the right place.
